static/src/javascripts/projects/common/modules/onward/more-tags.js

```diff
@@ -9,15 +9,16 @@ define([
     $,
     bean
 ) {
+    var ACTIVE_STATE = 'is-available';
 
     function MoreTags() {
         this.init = function () {
             var $more = $('.js-more-tags');
             if ($more.length !== 0) {
-                $more.removeClass('modern-hidden');
+                $more.addClass(ACTIVE_STATE);
                 bean.on(document.querySelector('.js-more-tags__link'), 'click', function () {
                     $('.modern-hidden-tag').removeClass('modern-hidden');
```

Comment:
this should probably be scoped or maybe changed to a more bem style selector.  Futhermore, for ones used in js, it should start with js-.  If you had time to tidy it it might be worth it.